package.json

```diff
@@ -1281,6 +1292,49 @@
           "type": "integer",
           "default": 1,
           "description": "Number of header rows to assume. Set to 0 to disable.`"
+        },
+        "latex-workshop.formattedPaste.image.method": {
+          "type": "string",
+          "enum": [
+            "leave",
+            "copy",
+            "move"
+          ],
+          "enumDescriptions": [
+            "reference the image where it is",
+            "copy image to target location",
+            "move image to target location"
+          ],
+          "default": "leave",
+          "description": "When pasting the path to an image, determine how it is treated. With copy and move 'target location' is graphicsdir or cwd"
+        },
+        "latex-workshop.formattedPaste.image.template": {
+          "type": [
+            "string",
+            "array"
+          ],
+          "default": "\\includegraphics{${imageFilePath}}",
+          "description": "The template to use for pasting the image (use an array for multiline strings), use ${imageFilePath} to represent the image file. Can also use ${imageFileNameWithoutExt}"
+        },
+        "latex-workshop.formattedPaste.image.defaultName": {
+          "type": "string",
+          "default": "Y-MM-DD-HH-mm-ss",
+          "description": "The default copied image file name. The value of this config will be pass to the 'format' function of moment library (a js time manipulation library), you can read document https://momentjs.com/docs/#/displaying/format/ for advanced usage. You can include unformatted text by enclosing with square brackets [like so]."
+        },
+        "latex-workshop.formattedPaste.image.filePathConfirmInputBoxMode": {
```

Comment:
Also remove this one and show relative path related to the root of project on clipboard case.

Reply:
Seems reasonable. I'll set the name to `image<number>`. Done in 51c026ab907634c1401d7fe18a1de974f5c1bc90

Reply:
Ooops, wrong comment. This is now done in d775384bf69c0cb0ac049c3f7e12a94ffbd6d2ae